plugin-api/src/main/java/org/hyperledger/besu/plugin/data/BlockHeader.java

```diff
@@ -154,4 +154,11 @@
    * @return The Keccak 256-bit hash of this header.
    */
   Hash getBlockHash();
+
+  /**
+   * The BASEFEE of this header.
+   *
+   * @return TheBASEFEE of this header.
+   */
+  Long getBaseFee();
```

Comment:
* Should be tagged `@Unstable`
* Should be a default method, returning `null`
* Also, perhaps `Optional<Long>` instead of just `Long`?  Always empty when the BASEFEE isn't relevant?  If so the default is `Optional.empty()`

Reply:
Done. Added `Unstable` annotation and moved to `Optional<Long>`with default `Optional.empty()`